app/views/policy/policy.njk

```diff
@@ -0,0 +1,29 @@
+{% extends "../layout.njk" %}
+
+{% block pageTitle %}
+   Policy Document Download - GOV.UK Pay
+{% endblock %}
+
+{% block mainContent %}
+<div class="govuk-grid-column-full">
+  <h1 class="govuk-heading-l page-title">
```

Comment:
`page-title` is dotted around the place because lots of tests look for it, if you’re not using it for tests, you dont need it